docker/1.14/Dockerfile.gpu

```diff
@@ -0,0 +1,127 @@
+FROM nvidia/cuda:10.0-base-ubuntu16.04
+
+LABEL maintainer="Amazon AI"
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+
+ENV NCCL_VERSION=2.4.7-1+cuda10.0
+ENV CUDNN_VERSION=7.5.1.10-1+cuda10.0
+ENV TF_TENSORRT_VERSION=5.0.2
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+        ca-certificates \
+        cuda-command-line-tools-10-0 \
+        cuda-cublas-10-0 \
+        cuda-cufft-10-0 \
+        cuda-curand-10-0 \
+        cuda-cusolver-10-0 \
+        cuda-cusparse-10-0 \
+        libcudnn7=${CUDNN_VERSION} \
+        libnccl2=${NCCL_VERSION} \
+        libgomp1 \
+        curl \
+        git \
+        wget \
+        vim \
+        #next two lines are needed to add python-3.6 should be removed from ubuntu-16.10
+        build-essential \
+        zlib1g-dev \
+        && \
+        apt-get clean && \
+        rm -rf /var/lib/apt/lists/*
+
+# The 'apt-get install' of nvinfer-runtime-trt-repo-ubuntu1604-4.0.1-ga-cuda10.0
+# adds a new list which contains libnvinfer library, so it needs another
+# 'apt-get update' to retrieve that list before it can actually install the
+# library.
+# We don't install libnvinfer-dev since we don't need to build against TensorRT,
+# and libnvinfer4 doesn't contain libnvinfer.a static library.
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends nvinfer-runtime-trt-repo-ubuntu1604-${TF_TENSORRT_VERSION}-ga-cuda10.0 && \
+    apt-get update && \
+    apt-get install -y --no-install-recommends libnvinfer5=${TF_TENSORRT_VERSION}-1+cuda10.0 && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/* && \
+    rm /usr/lib/x86_64-linux-gnu/libnvinfer_plugin* && \
+    rm /usr/lib/x86_64-linux-gnu/libnvcaffe_parser* && \
+    rm /usr/lib/x86_64-linux-gnu/libnvparsers*
+
+
+# See http://bugs.python.org/issue19846
+ENV LANG C.UTF-8
+
+# Add arguments to achieve the version, python and url
+#  PYTHON=python for 2.7
+#  PYTHON=python3 for 3.5, 3.6 is not available directly on 16.04
+ARG PYTHON=python3
+
+# user python-pip or python3-pip
+ARG PYTHON_PIP=python3-pip
+
+#  PIP=pip for 2.7
+#  PIP=pip3 for 3.5, 3.6 is not available directly on 16.04
+ARG PIP=pip3
+ARG PYTHON_VERSION=3.6.6
+
+RUN wget https://www.python.org/ftp/python/$PYTHON_VERSION/Python-$PYTHON_VERSION.tgz && \
```

Comment:
rm -rf /var/lib/apt/lists/*

Also, can we follow same indentation format for all dockerfiles?

combine ENV and ARGS, move to top of file etc